src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx

```diff
@@ -476,6 +480,14 @@ const FeatureFlagsPage: React.FC = () => {
     setLoading(flagsLoading || envsLoading)
   }, [flagsLoading, envsLoading])
 
+  useEffect(() => {
+    const params = {
+      ...queryParams,
+      ...(flagFilter && { [`${flagFilter.key}`]: flagFilter.value })
```

Comment:
Does this need to be a template string?

```suggestion
      ...(flagFilter && { [flagFilter.key]: flagFilter.value })
```

Reply:
I think it does when you're writing dynamic object key names, will try it without and see

Reply:
@chrisblakely01 turns out it doesn't need to be a string, will update